frontend/packages/kubevirt-plugin/integration-tests/tests/utils/mocks.ts

```diff
@@ -3,6 +3,8 @@ import { CloudInitConfig } from './types';
 import { STORAGE_CLASS } from './consts';
 import { getRandomMacAddress } from './utils';
 
+export const COMMON_TEMPLATES_VERSION = 'v0.6.2';
```

Comment:
maybe it would look better in `./consts`

Reply:
Done